test/helpers/cilium.go

```diff
@@ -878,7 +878,7 @@ func (s *SSHMeta) ServiceDelAll() *CmdRes {
 func (s *SSHMeta) SetUpCilium() error {
 	template := `
 PATH=/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/sbin:/sbin:/bin
-CILIUM_OPTS=--kvstore consul --kvstore-opt consul.address=127.0.0.1:8500 --debug --pprof=true --log-system-load --tofqdns-enable-poller=true
+CILIUM_OPTS=--kvstore consul --kvstore-opt consul.address=127.0.0.1:8500 --debug --pprof=true --log-system-load
```

Comment:
This means we no longer will have integration test coverage for the poller. cc: @raybejjani is there a plan to deprecate the poller eventually?

Reply:
poller is tested on `test/runtime/fqdn.go` no?

Reply:
@eloycoto yes, but I can't see an option to enable the poller, only ones to disable it. I think we assumed it was default on in the tests, and this will flip it around.

@ianvernon I'm not sure. I thought we would but there are a few use-cases where it is useful. My impression is that it will stay.

Reply:
@eloycoto yes, what @raybejjani said.

We should definitely have at least one test that ensures that the poller is working if we are going to merge this change.